ansible/group_vars/do/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+droplet_size: "s-1vcpu-2gb"
+main_node_size: "s-1vcpu-2gb"
+crawl_node_size: "c-4"
+droplet_region: "sfo3"
+do_project: "btrixcloud-{{ my_project | default('project')}}"
+space_name: "btrixcloud-{{ my_space_name | default('space')}}"
+btrix_db: "btrixcloud-{{ my_db | default('db')}}"
```

Comment:
if this was mit let it say `mit-project` `mit-space` `mit-db` 

`{{ space_name | default('btrixcloud') }}-bucket`
`{{ db_name | default('btrixcloud') }}-db`